test/extensions/filters/http/lua/lua_integration_test.cc

```diff
@@ -305,11 +316,10 @@ name: lua
 
   initializeFilter(FILTER_AND_CODE);
   codec_client_ = makeHttpConnection(makeClientConnection(lookupPort("http")));
-  Http::TestRequestHeaderMapImpl request_headers{{":method", "POST"},
-                                                 {":path", "/test/long/url"},
-                                                 {":scheme", "http"},
-                                                 {":authority", "host"},
-                                                 {"x-forwarded-for", "10.0.0.1"}};
+  Http::TestRequestHeaderMapImpl request_headers{
+      {":method", "POST"},     {":path", "/test/long/url"},     {":scheme", "http"},
+      {":authority", "host"},  {"x-forwarded-for", "10.0.0.1"}, {"x-test-header", "foo"},
+      {"x-test-header", "bar"}};
```

Comment:
Can we also test `cookie` header as well here, so we are sure when it is inlined (separated by ";", since cookie is special, and another inlined header separated by ",")?

Reply:
Added